src/applications/find-forms/containers/SearchForm.jsx

```diff
@@ -46,7 +67,12 @@ export class SearchForm extends Component {
 
   onSubmitHandler = event => {
     event.preventDefault();
-    this.props.fetchFormsThunk(this.state.query);
+    this.props.fetchFormsThunk(this.state.query).then(forms => {
+      // Derive the total number of pages.
+      const totalPages = Math.ceil(forms.length / MAX_PAGE_LIST_LENGTH);
+
+      return this.recordGAEvent(this.state.query, forms.length, totalPages);
+    });
```

Comment:
Same as other comment - if we could do this from the Redux action, then we can save on some repetitious code and wouldn't need the [eslint-override](https://github.com/department-of-veterans-affairs/vets-website/pull/15520/files#diff-4deb3f5adeff7ef1d02f9dc276c2270e5dc8bc0800bb927e3169eaad9d0bfc9dR76)

Reply:
Oh yeah, that's a good abstraction. Can do that.